README.md

```diff
@@ -38,16 +37,18 @@ Install the Cloud Foundry command line executable. You can find this on the [rel
 
 ### Push your app
 
-We push an mda (Mendix Deployment Archive) that was built by the Mendix Business Modeler to Cloud Foundry.
+We push an MDA (Mendix Deployment Archive) that was built by the Mendix Business Modeler to Cloud Foundry.
+
+*Note: please replace `<LINK-TO-BUILDPACK>` in the commands below with a link to the version of the buildpack you are trying to deploy. Please check [this section](#buildpack-releases-and-version-pinning) for details on how to pick a release.*
 
 ``` shell
-cf push <YOUR_APP> -b https://github.com/mendix/cf-mendix-buildpack -p <YOUR_MDA>.mda -t 180
+cf push <YOUR_APP> -b <LINK-TO-BUILDPACK> -p <YOUR_MDA>.mda -t 180
 ```
 
-We can also push a project directory. This will move the build process (using mxbuild) to Cloud Foundry:
+We can also push a project directory. This will move the build process (using MxBuild) to Cloud Foundry:
 
 ``` shell
-cd <PROJECT DIR>; cf push -b https://github.com/mendix/cf-mendix-buildpack -t 180
+cd <PROJECT DIR>; cf push -b <LINK-TO-BUILDPACK> -t 180
 ```
```

Comment:
Explanation is clear to me, but a bit abstract. In my own experience with open source projects, users may get confused if we don't give any concrete examples. This saves us some headaches.

Reply:
There's an explanation right at the top explaining how to substitute that in a command.
Also, I've added a disclaimer that users should at least be familiar with the Cloud Foundry CLI.